apps/web/modules/bookings/views/bookings-single-view.tsx

```diff
@@ -466,7 +464,7 @@ export default function Success(props: PageProps) {
                             ? t("no_longer_attending")
                             : t("event_cancelled")
                           : isBookingInPast
-                          ? t("event_expired")
+                          ? t("event_is_in_the_past")
```

Comment:
```
 {needsConfirmation && isReschedulable
                          ? props.recurringBookings
                            ? t("booking_submitted_recurring")
                            : t("booking_submitted")
                          : isCancelled
                          ? seatReferenceUid
                            ? t("no_longer_attending")
                            : t("event_cancelled")
                          : isBookingInPast
                          ? t("event_expired")
                          : props.recurringBookings
                          ? t("meeting_is_scheduled_recurring")
                          : t("meeting_is_scheduled")
```         
                
can we simplify this code?

Reply:
Yeah seems like a good opportunity to make it more readable. Will take care of this 🙏